pkg/app/mysql.go

```diff
@@ -0,0 +1,244 @@
+// Copyright 2019 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"time"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/format"
+	"github.com/kanisterio/kanister/pkg/helm"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+	"k8s.io/client-go/kubernetes"
+)
+
+type chartInfo struct {
+	helmrepoURL   string
+	helmChartName string
+	helmAppname   string
+}
+
+type MysqlDB struct {
+	cli       kubernetes.Interface
+	namespace string
+	password  string
+	chart     chartInfo
+}
+
+func NewMysqlDB(helmrepoURL, helmChart, helmAppname string) App {
+	return &MysqlDB{
+		password: "mysecretpassword",
+		chart: chartInfo{
+			helmrepoURL:   helmrepoURL,
+			helmChartName: helmChart,
+			helmAppname:   helmAppname,
+		},
+	}
+}
+
+func (mdb *MysqlDB) Init(ctx context.Context) error {
```

Comment:
@viveksinghggits We should check if required AWS creds are set in Init(). If not return error to skip the test